README.md

```diff
@@ -42,7 +42,7 @@ yum -y install xfsprogs                # Only if XFS file system is required
 #### 2. Configure Linux® multipath devices on the host.
 
 **Important:** Be sure to configure each worker with storage connectivity according to your storage system instructions. 
-For more information, find your storage system documentation on [IBM Knowledge Center](https://www.ibm.com/support/knowledgecenter).
+For more information, find your storage system documentation on [IBM Knowledge Center](https://www.ibm.com/docs/en).
```

Comment:
find your storage system product information in [IBM Documentation](https://www.ibm.com/docs/en/).

Reply:
done